test('Hello world !',()=>{
    console.log(`Hey!`);
});
test('error',()=>{
    throw new Error(`Failure!`);
})